frontend/src/App.tsx

```diff
@@ -10,7 +10,7 @@ import LogOut from './components/Authentication/LogOut.tsx';
 import UserAccount from './pages/UserAccount.tsx';
 import UserProfile from './pages/UserProfile.tsx';
 import Chat from './pages/Chat.tsx';
-// import GameApp from './pages/game';
+import Game from './pages/game.tsx';
```

Comment:
Beetje nitpicky dit maar een style dingetje: Het is denk ik mooi als alle files een capitalized naam hebben. Dus Game ipv game. Dan past het bij hoe de files in main er nu uit zien.